packages/fxa-payments-server/src/components/PaymentErrorView/index.tsx

```diff
@@ -153,13 +153,16 @@ export const PaymentErrorView = ({
           <PaymentErrorMessage />
         </div>
 
-        <div className="footer" data-testid="footer">
-          {/* This error code means the subscription was created successfully, but
+        {/* This error code means the subscription was created successfully, but
           there was an error loading the information on the success screen. In this
           case, we do not want a "Try again" or "Manage subscription" button. */}
-          {error.code !== 'fxa_fetch_profile_customer_error' ? (
+        {error.code !== 'fxa_fetch_profile_customer_error' && (
+          <div className="options mb-6" data-testid="options">
```

Comment:
Added to be consistent with how the button is styled in `PaymentConfirmation`

Reply:
Good catch! Thank you for adding.

Adding a quick note here that this will be revisited in FXA-5620 and/or FXA-5637. `options mb-6` will likely be replaced with `button-row` to remain consistent with other routes such as `Checkout`, `SubscriptionCreate`, and `SubscriptionUpgrade`. `options` is declared in `checkout-view()` and appears to only be used these two files.